frontend/src/utils.ts

```diff
@@ -63,3 +63,28 @@ export const sortListByPopularity = (list: object[]) => {
 export const uniqueArray = (list: object[], filterBy: string) => {
   return [...new Map(list.map(item => [item[filterBy], item])).values()]
 }
+
+export const calculateAmountOfShownItems = (
+  viewportWidth: number,
+  amounts: number[]
```

Comment:
I'd love if the amounts were interfaced with a `viewportAmount` and use it as an object instead of a list

Reply:
Done